backend/tracim_backend/lib/search/elasticsearch_search/es_models.py

```diff
@@ -34,27 +34,54 @@
 edge_ngram_token_filter = analysis.token_filter(
     "edge_ngram_filter", type="edge_ngram", min_ngram=2, max_gram=20
 )
+# NOTE 2021-11-02 - S.G. - Configuring a maximum length for tokens (e.g. words)
+# so that the following error does not occur:
+# https://stackoverflow.com/questions/24019868/utf8-encoding-is-longer-than-the-max-length-32766
+# It can happen in HTML custom properties as images/video are embedded in it
+# when the custom properties schema does not set '"format"="html"'.
+max_token_length_filter = analysis.token_filter(
+    "max_token_length_filter", type="length", min=0, max=32766
+)
+# NOTE 2021-11-04 - G.M. - Configuring a maximum length for keyword:
+# like tokens, keyword bigger than 32766 will failed, this is the max size accepted for keyword,
+# if text is bigger, keyword will not be stored.
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html
+# ignore above count by character, so for utf-8 8191*4 ( 4 is max byte for utf-8 character) = 32766
```

Comment:
I suggest to open an issue to filter out images and video so most profiles can be indexed even if they contain images and videos

Reply:
Image and video are already filtered if the type is html (the html is removed , only the text stay), error with too big token happened in custom properties where type is not set as html but there is still tinymce.

This fix is mostly a fallback to support some edge case that have little chance to happened unless custom properties are not correctly configured or something put very weird content in tracim.